src/config/swagger.yaml

```
@@ -0,0 +1,272 @@
+                    error:
+                      type: string
+                      description: Error message
+                description: List of errors encountered during eligibility check
+        '400':
+          description: Bad request
+          schema:
+            type: object
+            properties:
+              error:
+                type: string
+                description: Error message 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)